src/Nethermind/Nethermind.Blockchain/Synchronization/ISyncConfig.cs

```diff
@@ -78,6 +78,9 @@ public interface ISyncConfig : IConfig
         [ConfigItem(Description = "Enables SNAP sync protocol.", DefaultValue = "false")]
         public bool SnapSync { get; set; }
 
+        [ConfigItem(Description = "Number of account range partition to create. Increase snap sync request concurrency. ", DefaultValue = "8")]
+        int SnapSyncAccountRangePartitionCount { get; set; }
+
```

Comment:
Would it make sense to default to Environment.ProcessorCount?

Reply:
No I dont think it work that way. Think of it as num of task spawner instead of num of task. For the record, the last run for the verify trie uses 64 partition. Does not really do much.